oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elasticsearch/ElasticsearchIndexProviderService.java

```diff
@@ -107,6 +107,16 @@
     )
     private static final String PROP_ELASTICSEARCH_PORT = ElasticsearchConnection.PORT_PROP;
 
+    @Property(
+            label = "Elasticsearch API key ID"
+    )
+    private static final String PROP_ELASTICSEARCH_API_KEY_ID = ElasticsearchConnection.API_KEY_ID_PROP;
+
+    @Property(
+            label = "Elasticsearch API key secret"
+    )
+    private static final String PROP_ELASTICSEARCH_API_KEY_SECRET = ElasticsearchConnection.API_KEY_SECRET_PROP;
```

Comment:
I think there is a way to obscure this Secret (I think it was by using default passwordValue). Currently the API secret key would be visible in the config manager and also if someone exports the config.
Possibly there could be some other annotation as well in Felix for handling passwords/secrets.

Reply:
It should be OK now.